deployer/scripts/upgrade.sh

```diff
@@ -46,6 +46,12 @@ function getDeploymentVersion() {
     return
   fi
 
+  # check for common data model
+  if [[ -n "$(oc get configmap/logging-elasticsearch -o yaml | grep 'use_common_data_model: true')" ]]; then
```

Comment:
If someone decides not to use the common data model, then `$(oc get configmap/logging-elasticsearch -o yaml | grep 'use_common_data_model: true')` will return nothing (the empty string), which means the `if [[ -n "" ]]` will return false, which means the `echo 6` will be skipped, which means the script will fall through to `echo "$LOGGING_VERSION"`